docs/AboutUs.md

```diff
@@ -5,7 +5,13 @@ title: About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You may reach us through the following email addresses:
+
+Joe Teddy Yosua Sanjaya: joe.teddy@u.nus.edu
+Yu Bowei: e0425419@u.nus.edu
+Eddy Chu: e0418218@u.nus.edu
+Ngo Duc Binh: e0271216@u.nus.edu
+Petrick Jerico Gustendi: e0425802@u.nus.edu
```

Comment:
Would be good if we can make all 5 emails in the same format, but that's very minor. Just a thought!